src/commands/account/delete_account/mod.rs

```diff
@@ -152,7 +152,14 @@ impl BeneficiaryAccount {
                 )
                 .is_none()
                 {
-                    eprintln!("\nHeads up! You will lose remaining NEAR tokens on the account you delete if you specify the account <{account_id}> as the beneficiary as it does not exist.");
+                    let networks: Vec<String> = context
+                        .global_context
+                        .config
+                        .network_connection
+                        .iter()
+                        .map(|(_, network_config)| network_config.network_name.clone())
+                        .collect();
```

Comment:
Make it a method on `config`, so you can just call it like:

```
global_context.config.network_names()
```

Use it in every place where you currently copied this code manually.